server/broadcast_test.go

```diff
@@ -393,9 +546,10 @@ func TestRefreshSessions(t *testing.T) {
 	sess3 := StubBroadcastSession("transcoder3")
 	sess4 := StubBroadcastSession("transcoder4")
 
-	bsm.createSessions = func() ([]*BroadcastSession, error) {
-		return []*BroadcastSession{sess3, sess4}, nil
-	}
+	bsm.sessMap[sess3.OrchestratorInfo.Transcoder] = sess3
```

Comment:
This code directly adds new sessions to sessMap/sessList and then we check if these sessions are properly added to sessMap/sessList. This is a different from the old test which was checking if given that `createSessions()` returns sess3 and sess4 whether the sessMap/sessList will be properly updated after a call to `refreshSessions()`.

Reply:
fixed in 2c78fb9a784ea81c42a9bdbd0d91c71ab6fc255b